package.json

```diff
@@ -18,7 +18,7 @@
   "license": "ISC",
   "readmeFilename": "README.md",
   "dependencies": {
-    "nan": "^2.11.1",
+    "node-addon-api": "1.6.0",
```

Comment:
This should be `^1.6.0` so that, when node-addon-api@1.7.0 comes out this package will start using it.